_sass/table.scss

```diff
@@ -0,0 +1,45 @@
+//-------------------- General definitions --------------------//
+$font-stack: 'Rubik','Roboto' , sans-serif; //A list of fonts
+$table-w: 100%; // width definition , relative to parent
+
+//-------------------- Colors -------------------- //
+$header-bg-color: #4563dd; //header color
+$header-text-color: #FFF; //header text color
+$table-text-color: #000; //table text color
+$border-style: 1px solid #CCC; //simple grey border
+$row-primary-color: #f2f2f2;
+
+//-------------------- Utility --------------------//
+$text-align: left; // alignment , when you need it
+$skinny-font: 100; // fat free font size
+$small-screen: 800px; //used for media queries
+
+.table {
+  font-family: $font-stack;
+  color: $table-text-color;
+  border-collapse: collapse;
+  width: $table-w;
+  border: $border-style;
+  @media screen and (max-width: $small-screen) {
+    font-size: 0.8rem;
+  }
+
+  td,
+  th {
+    padding: 1rem;
+    border: $border-style;
+  }
+
+  th {
+    text-align: $text-align;
+    background-color: $header-bg-color;
+    font-weight: $skinny-font;
+    color: $header-text-color;
+  }
+
+  tr:nth-child(even) {
+    background-color: $row-primary-color;
+  }
+  //tr:nth-child(odd){background-color: $theme;} // Odd row color if needed. Watch out for value
```

Comment:
This bit is commented-out, is it still needed?

Reply:
No its not needed. Its just there for optional styling. It can be deleted, It just there in case you need to color "odd" table rows. I will sign off the commit during the day. :)